.github/workflows/extensions.yml

```diff
@@ -22,7 +22,9 @@ jobs:
     - name: Build
       shell: cmd
       run: |
-        cd extensions/build
+        cd extensions
+        mkdir build
```

Comment:
Why? This folder is Git tracked.

Reply:
changed build folders to 
extensions/vcproj32
extensions/vcproj64